api/mapping/admin.py

```diff
@@ -1,10 +1,23 @@
 from django.contrib import admin
 
-from .models import DocumentType, Source, DataPartner, ScanReport, \
-    ScanReportTable, \
-    ScanReportField, ScanReportValue, ScanReportAssertion, \
-    ClassificationSystem, OmopTable, OmopField, Document, \
-    DocumentFile, DataDictionary, StructuralMappingRule
+from .models import (
```

Comment:
is this after isort?

Reply:
Nope, just PEP8. I _think_ isort also puts imports into alphabetical order.